sdk/script/acquisition-sdk.ts

```diff
@@ -134,25 +148,37 @@ export class AcquisitionManager {
         });
     }
 
-    public reportStatus(status: string, message?: string, callback?: Callback<void>): void {
-        var url: string = this._serverUrl + "reportStatus";
-        var body: string;
-
-        switch (status) {
-            case AcquisitionStatus.DeploymentSucceeded:
-            case AcquisitionStatus.DeploymentFailed:
-                url += "/deploy";
-                body = JSON.stringify({ deploymentKey: this._deploymentKey, status: status, message: message });
-                break;
-
-            default:
-                if (callback) {
-                    callback(new Error("Unrecognized status."), /*not used*/ null);
-                }
-                return;
+    public reportStatus(package?: Package, status?: string, callback?: Callback<void>): void {
```

Comment:
I might be wrong, but as long as the semver version is bumped appropriately I think he can change these methods how he pleases, because the SDK version referenced by the client plugins is fixed.